src/App.js

```diff
@@ -1,25 +1,6 @@
-import logo from './logo.svg';
 import './App.css';
+import Greeting from './components/Greeting';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+const App = () => <Greeting />;
```

Comment:
- [ ] For this file, it's expected that a route is present that routes to the Greeting component and is done with the help of react-router-dom. Kindly set up react-router and add a link that routes to the Greeting component from the App.js 👍🏽

<img width="1079" alt="Screenshot 2023-07-11 at 10 23 28 AM" src="https://github.com/AmineDerbal/Hello-front-end-app/assets/50941074/2c9f9db0-8f6d-4d61-a761-0b1ea449acc2">

Reply:
@AmaduKamara 
Understood I will fix this 👍 
Thanks for your feeback 😄